vscode-ext/src/tests/e2e.ts

```diff
@@ -0,0 +1,146 @@
+            if (mirrordSettingsButton !== undefined) {
+                mirrordSettingsButton.click();
+                return true;
+            }
+        }, DEFAULT_CONDITION_TIMEOUT, "mirrord config `$(gear)` button not found -- timed out");
+
+        await browser.driver.wait(async () => {
+            return await existsSync(mirrordConfigPath);
+        }, DEFAULT_CONDITION_TIMEOUT, "mirrord `default` config not found");
+    });
+
+    it("select pod from quickpick", async function () {
+        await setBreakPoint(fileName, breakPoint, browser);
+        await startDebugging(configurationFile);
+
+        const inputBox = await InputBox.create();
+        // assertion that podToSelect is not undefined is done in "before" block   
+        await browser.driver.wait(async () => {
+            return await inputBox.isDisplayed();
+        }, DEFAULT_CONDITION_TIMEOUT, "quickPick not found -- timed out");
+        await inputBox.selectQuickPick(podToSelect!);
+    });
+
+    it("wait for breakpoint to be hit", async function () {
+        debugToolbar = await DebugToolbar.create(20000);
+        // waiting for breakpoint and sending traffic to pod are run in parallel
+        // however, traffic is sent after 10 seconds that we are sure the IDE is listening
+        // for breakpoints
+        await browser.driver.wait(async () => {
+            return await debugToolbar.isDisplayed();
+        }, 20000, "debug toolbar not found -- timed out");
+
+        await Promise.all([debugToolbar.waitForBreakPoint(), sendTrafficToPod()]);
+    });
+});
+
+// This promise is run in parallel to the promise waiting for the breakpoint to be hit
+// We wait for 10 seconds to make sure that we are in listening state
+async function sendTrafficToPod() {
+    await new Promise(resolve => setTimeout(resolve, 10000));
+    const response = await get(kubeService!!);
+    expect(response.status).to.equal(200);
+    expect(response.data).to.equal("OK - GET: Request completed\n");
+}
+
+// opens and sets a breakpoint in the given file
+async function setBreakPoint(fileName: string, breakPoint: number, browser: VSBrowser, timeout: number = 10000) {
+    const editorView = new EditorView();
+    await editorView.openEditor(fileName);
+    const currentTab = await editorView.getActiveTab();
+    expect(currentTab).to.not.be.undefined;
+    await browser.driver.wait(async () => {
+        const tabTitle = await currentTab?.getTitle();
+        if (tabTitle !== undefined) {
+            return tabTitle === fileName;
+        }        
+    }, timeout, "editor tab title not found -- timed out");
+
+    const textEditor = new TextEditor();
+    const result = await textEditor.toggleBreakpoint(breakPoint);
+    expect(result).to.be.true;
+}
+
+// starts debugging the current file with the provided configuration
+// debugging starts from the "Run and Debug" button in the activity bar
+async function startDebugging(configurationFile: string) {
```

Comment:
```suggestion
async function startDebugging(configurationFile: string = "Python: Current File") {
```

And remove the const